tests/flow/test_udf.py

```
@@ -0,0 +1,1035 @@
+        self.env.assertTrue(v is not None)
+
+        # RegExp → converted to string
+        v = self.graph.query("RETURN ReturnRegExp()").result_set[0][0]
+        self.env.assertEqual(v, "/abc.*/")
+
+        # Symbol is not supported
+        try:
+            v = self.graph.query("RETURN ReturnSymbol()").result_set[0][0]
+            self.env.assertTrue(False)
+        except Exception:
+            pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)